### PR TITLE
changes needed for encrypted assertions

### PR DESCRIPTION
**Pull request by gerbsen**
So, I just got the changes from @anicoa and I integrated them in a fresh `develop` branch. Since it was a little bit difficult to actually merge since due to tabs/spaces/newlines etc. I would like to have @anicoa to have a look at it and give his go (he's the one who develop the fix). In the meantime you could check if this fits with your standards!

### CODE REVIEW

**Comment by gerbsen** [comment]
Okay, we've testet this on our side. And we think it's good to go. What do you think?

**Comment by steffow** [comment]
I’ll have look. I’m a bit busy at the moment, but will come back to you.

**Comment by gerbsen** [comment]
Any chance yet to look at it?

File: saml_utils.js [hunk 1]
```diff
@@ -374,6 +374,17 @@ SAML.prototype.validateResponse = function(samlResponse, relayState, callback) {
                 }
 
                 const assertion = response.getElementsByTagNameNS('urn:oasis:names:tc:SAML:2.0:assertion', 'Assertion')[0];
+                const encAssertion = response.getElementsByTagNameNS('urn:oasis:names:tc:SAML:2.0:assertion', 'EncryptedAssertion')[0];
+
+                var xmlenc = require('xml-encryption');
+                var options = { key: this.options.privateKey, };
```
Comment (steffow) [inline saml_utils.js]: What are we doing after the comma?

Comment (gerbsen) [reply saml_utils.js]: nothing, I just removed it

**Comment by gerbsen** [comment]
Okay, I added a section for encryption now. Does that make sense?

**Comment by gerbsen** [comment]
@steffow from our side this is done. okay to merge?

**Review by steffow — APPROVED** [review]

**Review by steffow — APPROVED** [review]

**Comment by steffow** [comment]
Thanks. Published to Atmosphere. Thanks also for bearing w my time management ;-)

**Pull request merged**